Update ISSUE_TEMPLATE.md (#46)
Carvera-Community/Carvera_Community_Firmware: ISSUE_TEMPLATE.md

## How to file an issue

> If you encounter an issue Carvera Community Firmware, you are welcome to

Reviewer: ```suggestion
If you encounter an issue using the Carvera Community Firmware, you are welcome to
```

### What should I do before submitting an issue?

> 1. make sure you are running the community firmware and controller and check through the change notes to see if it has already been addressed. 

Reviewer: ```suggestion
1. Make sure you are running the latest released community firmware and controller and check through the change notes to see if your issue has already been addressed. 
```

#### What happened instead?

#### Branch & Commit or Version of The firmware and controller

    [Can be found with the version command. (@version in the controller MDI)]

#### Operating System

#### Any MDI output or screenshots that help explain the issue

> Copy-paste this template **completely**. Do not skip any lines!

Reviewer: ```suggestion
Please fill out this template **completely**. Do not skip any lines!
```
The template is used as the default content in a new issue.
